src/scss/settings/_variables.scss

```diff
@@ -20,6 +20,9 @@ $type: (
     base: (  // body copy
         default: (14, 18)
     ),
+    mid--scaleDown: (
```

Comment:
Would it be better to name this as a `base--scaleUp `, just as it relates closer to the base sizing than `mid`?